docs/source/index.rst

```diff
@@ -58,15 +58,15 @@ Stable Packages
 +--------------------------------------------+--------------------+--------+
 | Daml.Finance.Claims                        | 1.0.1              | Stable |
 +--------------------------------------------+--------------------+--------+
-| Daml.Finance.Data                          | 1.0.1              | Stable |
+| Daml.Finance.Data                          | 2.0.0              | Stable |
```

Comment:
I wonder if this is considered a major change to the `Daml.Finance.Data` package.

We are not touching any choice within a template, I would almost consider this to be a patch fix. @georg-da any opinion on this?